doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -987,6 +987,8 @@ interpreted in the scope stack extended with the scope bound tokey.
    To bind a delimiting key to a scope, use the command
    :n:`Delimit Scope @scope with @ident`
 
+   .. warn:: Overwriting previous delimiting key @ident in scope @scope
```

Comment:
Add `:undocumented:` on the next line (to replace the missing description). We are trying to avoid all objects without a body but we can't turn the warning on yet because there are many remaining.